src/quacc/recipes/espresso/phonons.py

```diff
@@ -56,10 +85,192 @@ def phonon_job(
     }
 
     return base_fn(
-        template=EspressoTemplate("ph"),
+        template=EspressoTemplate("ph", test_run=test_run),
         calc_defaults=calc_defaults,
         calc_swaps=calc_kwargs,
         parallel_info=parallel_info,
         additional_fields={"name": "ph.x Phonon"},
         copy_files=prev_dir,
     )
+
+
+# 1. The name seems confusing to me (this is grid parallelization),
+# overall I would argue to globally
+# change the name of these jobs to distinguish them from other kind of
+# phonon calculations. ph.x is not limited to phonon calculations after all.
```

Comment:
Agreed. Feel free to rename, perhaps to `_phonon_forces_subflow` like what is done [here](https://github.com/Quantum-Accelerators/quacc/blob/10f17dda01e0594204a82e89941f5d2c0f6ed539/src/quacc/recipes/common/phonons.py#L72-L82).

The `_` in the front indicates to the user that it is not meant to be used directly (except by expert users), and the recipe will not appear in the API documentation by default. This is intentional.

In fact, one could consider moving this function _inside_ of `grid_phonon_flow` --- the only reason I didn't is in case we need to use it for some other recipe later on.

Reply:
Ok, would it be possible to construct a custom flow with this subflow ? (I wanted to do a example in the doc)

Reply:
It is certainly possible, but I don't know that I'd recommend it for most users. What is the example you had in mind? If it's to call the grid phonon workflow after a prior job, then you can make `relax_job` an optional first step and then call the `grid_phonon_flow` as a second step in your custom workflow.

Reply:
Coming back quickly on that: how would you make the relax_job optional? I had a look at the doc but didn't find how to do it